app/furniture/marketplace/carts/_footer.html.erb

```diff
@@ -7,7 +7,7 @@
 <tfoot id="cart-footer-<%= cart.id%>" class="bg-gray-50">
   <tr>
     <td></td>
-    <th> <%= link_to("checkout", new_space_room_marketplace_checkout_path(space, room, marketplace), data: { turbo: false }) %> </th>
+    <th> <%= button_to("checkout", [space, room, marketplace, :checkouts], data: { turbo: false }) %> </th>
```

Comment:
Theoretically, `marketplace.location(:checkouts)` should return `[space, room, marketplace, :checkouts]`

Reply:
Noted. We can update it on Wednesday.

Reply:
That doesn't seem to work. `marketplace.location(:checkouts)` is creating the path `/spaces/system-test/rooms/entrance-hall/marketplaces/000c9416-a6ff-49f0-9a55-3a86aad16c3a`, which is not right.

Reply:
Blah, I must not have finished applying`HasLocation` to the `Marketplace` namespace.